src/e2e/resources/data/AdminSearchPageE2ETest_SqlEntities.json

```diff
@@ -0,0 +1,44 @@
+{
+  "accounts": {
+    "instructor1OfCourse1": {
+      "googleId": "tm.e2e.ASearch.instr1",
+      "name": "Instructor1 of Course1",
+      "email": "ASearch.instructor1@gmail.tmt",
+      "readNotifications": {}
```

Comment:
Should the `readNotifications` should be removed and replaced with an `id` field? Since those are the fields in the `Account` sql entity class